usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -94,15 +98,13 @@ def handle(self, *args, **options):
         loader = ThreadedDataLoader(Procurement, field_map=field_map, value_map=value_map)
         loader.load_from_file(options['file'][0])
 
-    def get_agency(self, row):
-        agency = Agency.objects.filter(subtier_code=self.get_agency_code(row['maj_agency_cat'])).first()
+    def get_agency(self, agency_string):
+        agency = Agency.objects.filter(subtier_code=self.get_agency_code(agency_string)).first()
```

Comment:
I think for some things, .first() makes sense. (Here, because agencies are guaranteed to be unique). The main reason is .filter().first() will return None if there are no results, whereas .get() will throw an exception. We'd just need a slightly more robust  try/catch block all over that place that might be worth putting somewhere it could be shared.